### PR TITLE
housekeeping

### PR DESCRIPTION
**Pull request by zarsky-broad**

### CODE REVIEW

File: integration/build.gradle [hunk 1]
```diff
@@ -12,17 +12,16 @@ compileJava { options.release = 11 }
 
 repositories {
 	mavenCentral()
-	maven { url 'https://broadinstitute.jfrog.io/broadinstitute/libs-snapshot-local/' }
-	maven { url 'https://broadinstitute.jfrog.io/broadinstitute/libs-release/' }
+	maven { url 'https://broadinstitute.jfrog.io/artifactory/libs-snapshot' }
```
Comment (zarsky-broad) [inline integration/build.gradle:15]: 1. `/artifactory/` is the new recommended url scheme, it's the same
2. the only things we use from `libs-release` is the sam client, so I only kept it there
3. `libs-snapshot` includes `libs-snapshot-local`, but it's recommended to use a virtual repository (apparently), which is what this is.

File: settings.gradle [hunk 2]
```diff
@@ -3,12 +3,12 @@ pluginManagement {
...
 			eachPlugin {
 				if (requested.id.id == 'bio.terra.test-runner-plugin') {
-					useModule("bio.terra:terra-test-runner:" + requested.version)
+					useModule("bio.terra:terra-test-runner:${requested.version}")
 				}
 			}
 		}
 		maven {
-			url "https://broadinstitute.jfrog.io/broadinstitute/libs-snapshot-local/"
+			url 'https://broadinstitute.jfrog.io/artifactory/plugins-snapshot'
```
Comment (zarsky-broad) [inline settings.gradle:11]: `plugins-snapshot` is a virtual repository that includes `libs-snapshot-local`

File: settings.gradle [hunk 3]
```diff
@@ -3,12 +3,12 @@ pluginManagement {
 		resolutionStrategy {
 			eachPlugin {
 				if (requested.id.id == 'bio.terra.test-runner-plugin') {
-					useModule("bio.terra:terra-test-runner:" + requested.version)
+					useModule("bio.terra:terra-test-runner:${requested.version}")
 				}
 			}
 		}
```
Comment (zarsky-broad) [inline settings.gradle:9]: This can all go away if they resolve https://broadinstitute.slack.com/archives/C020RSQF5KL/p1630534822020700

**Review by Shakespeared — APPROVED** [review]
LGTM

**Review by snwessel — APPROVED** [review]
Looks good!

**Pull request merged**